app/assets/javascripts/activeadmin_addons/select2.js

```diff
@@ -62,6 +62,17 @@ $(function() {
           width: 'resolve',
           allowClear: allowClear
         });
+
+        var $parent = $(el).parent(), $label = $parent.find('label');
```

Comment:
Unexpected var, use let or const instead                           no-var<br>All 'var' declarations must be at the top of the function scope    vars-on-top<br>Split 'var' declarations into multiple statements                  one-var<br>'$' is not defined                                                 no-undef<br>Expected variable declaration to be on a new line                  one-var-declaration-per-line